scania-digital-design-system/tegel
# fix(table): auto-collapse expanded row upon clicking another expandable row

## **Describe pull-request**  
This pull-request adds the possibility for the developer to enable "auto collapse". This means that if its enabled, all other rows except for the row selected will be closed.

## **Issue Linking:**  
_Choose one of the following options_
- **Jira:** [CDEP-3490](https://tegel.atlassian.net/browse/CDEP-3490)
- **GitHub:**  [Issue 379](https://github.com/scania-digital-design-system/tegel/issues/739)

## **How to test**  
_Provide detailed steps for testing, including any necessary setup._
1. Go to Storybook
2. Navigate to the table component 
3. Go to expandable rows
4. Toggle the control autoCollapse to true
5. Test open multiple rows now, the expected behavior is that all other rows expect the selected row should close


## **Checklist before submission**

## **Suggested test steps**



[CDEP-3490]: https://tegel.atlassian.net/browse/CDEP-3490?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
